### ISSUE TITLE
Some pages lags as you scroll up with Linkclump selection box and some makred links don't opened

### ISSUE THREAD

**Issue by egozi44** [benblack86/linkclump#227]
Hi there, I'm a long time user of multi link openers that were used in the same fashion of Linkclump (very first addon I've ever used was Multi Links Plus for old FireFox which imo was the best),
With that being said I've want to report about three issues as well as ask for some potential new feature that was presented on old plugins I've used before such as said Multi Links Plus.

First issue is:
that when you want to scroll up/down a page as you hold two keys (right mouse and left shift in my case) in order to open the selection box, the page will lag as it scrolling, that didn't happened to me even on other addons that were avaliable in the chorme store in the past such as Snap Links, the page would've always scroll fine,

Second issue in additonal to the first one above: 
it's seem that using the the keys won't make the broswer to ignore their initial uses, and by that I mean that if I use hotkeys I've set for Linkclump that are also doing something in the actual browser (being used as the browser hotkeys in other words), they will do both actions at times at once, when I use Shift and right mouse as hotkeys it will also mark some sections in the page with blue color cause the site think I'm also trying to use shift to select site elements that got nothing to do with Linkclump, or so I'm assume (?)

Third issue is:
that if you're using two hotkeys, Linkclump won't always open the marked links you marked with the selection box since it seems to look for a very specific timing for the user to release the keys as you mark links, 
All of these 3 issues wasn't presented on old multi link opener addons and I really want to see them gone from Linkclump if possible :0

Now for the last thing which is me asking for some additonal feature over a bug fix,
In multi link plus for old firefox
I had the abiliy to hold two keys to select something, then release one key in order to keep move around the page with no selection box opened, and then re-press the same hotkey I've left to open a new selection box to select another links, all while still keeping the old selected fields still marked and then open the links on tabs once the I've released both keys (and only when I've released both keys)

In other words, when two keys held = you open selection box,

When one key is being held after the second one was released = you can now move around the page without keep using the old selection box *BUT* whatever you marked still marked to be opened once you leave both keys, 

When you hold both keys after releasing one of them and then re-hold it again = a new selection box is opened while still NOT canceling the old one, and still not open the old links you marked on new tabs,

Only when both keys are being selected and then released = All links you've marked so far will be opened in new tabs,

That's the gist of it, will it be possible to add something like that to Linkclump too?
It will make link selecting to be so much more efficient.
